### ISSUE TITLE
Debian package generation issue with absolute path

### ISSUE THREAD

**Issue by sweetim** [Cylix/cpp_redis#148]
Found these command in `CMakeLists.txt` at  `tacopie` and `cpp_redis` 

#### cpp_redis/CMakeLists.txt
```
# ensure lib and bin directories exist
install(DIRECTORY DESTINATION ${CMAKE_LIBRARY_OUTPUT_DIRECTORY})
install(DIRECTORY DESTINATION ${CMAKE_RUNTIME_OUTPUT_DIRECTORY})
```

#### tacopie/CMakeLists.txt
```
# ensure lib and bin directories exist
install(DIRECTORY DESTINATION ${CMAKE_BINARY_DIR}/lib/)
install(DIRECTORY DESTINATION ${CMAKE_BINARY_DIR}/bin/)
```

The above lines seems to request the creation of an absolute directory when trying to create a debian package with CPack.

What about using those commands instead to create the folder, this will prevent the absolute path to be installed on to the system:
```
file(MAKE_DIRECTORY ${CMAKE_LIBRARY_OUTPUT_DIRECTORY})
file(MAKE_DIRECTORY ${CMAKE_RUNTIME_OUTPUT_DIRECTORY})
```

**Comment by Cylix** [comment]
Hi,

I'll have some tests on my side to see if it breaks the compilation process (that CMake configuration is always such a pain in the ass :/).

Thanks for the suggestion!

**Comment by Cylix** [comment]
Hi,

Sorry for the loooong delay concerning this issue.
This one is finally fixed thanks to #159 (cpp_redis) and e8eaad3 (tacopie).

I'm closing this issue but do not hesitate to re-open it if necessary.

Best
